src/momento/config/transport/transport_strategy.py

```diff
@@ -46,15 +47,25 @@ def with_client_timeout(self, client_timeout: timedelta) -> TransportStrategy:
 
 
 class StaticGrpcConfiguration(GrpcConfiguration):
-    def __init__(self, deadline: timedelta):
+    _deadline: timedelta
+    _root_certificates: Optional[bytes]
```

Comment:
Would it make sense to have a separate gRPC configuration altogether for MVI as these don't seem related to cache? The only reason I'm saying is both these requirements may have divergent configurations over time, and I'm not sure you'd want to model all of them as `Optional` here because we don't want to break compatibility with the client configurations.